package.json

```diff
@@ -1,9 +1,13 @@
 {
+  "scripts": {
```

Comment:
Super nitpicky nit 😁: could you keep this file organized please? I like keeping my config files alphabetically sorted if possible (it makes it easier to search for things), and since this one otherwise already is...

Reply:
Yea I was wondering why the package name was down 😂😂. I'll refactor ⚡️